pop-api/primitives/src/lib.rs

```diff
@@ -1,3 +1,18 @@
 #![cfg_attr(not(feature = "std"), no_std, no_main)]
 
+pub use bounded_collections::{BoundedBTreeMap, BoundedBTreeSet, BoundedVec, ConstU32};
+//use scale::{Decode, Encode, MaxEncodedLen};
+
 pub mod storage_keys;
+
+// /// Some way of identifying an account on the chain.
+// #[derive(Encode, Decode, Debug, MaxEncodedLen)]
+// pub struct AccountId([u8; 32]);
```

Comment:
Remove comment if no longer necessary

Reply:
See comment in description about AccountId issues.